src/shared/parse.cpp

```diff
@@ -379,7 +379,7 @@ static void PRINTF_LIKE(2) Parse_ScriptError( const script_t *script, const char
 	if ( script->flags & SCFL_NOERRORS ) { return; }
 
 	va_start( ap, str );
-	vsprintf( text, str, ap );
+	vsnprintf( text, sizeof( text ), str, ap );
```

Comment:
You have to use Q_vsnprintf because it is broken on MSVC

Reply:
Done